integrationtests/agent/bundle_deployment_status_test.go

```diff
@@ -1,38 +1,38 @@
 package agent
 
 import (
+	"context"
 	"os"
 
+	"github.com/rancher/fleet/integrationtests/utils"
 	"github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func orphanBundeResources() map[string][]v1alpha1.BundleResource {
-	v1, err := os.ReadFile(assetsPath + "/deployment-v1.yaml")
-	Expect(err).NotTo(HaveOccurred())
-	v2, err := os.ReadFile(assetsPath + "/deployment-v2.yaml")
-	Expect(err).NotTo(HaveOccurred())
-
-	return map[string][]v1alpha1.BundleResource{
-		"v1": {
-			{
-				Name:     "deployment-v1.yaml",
-				Content:  string(v1),
-				Encoding: "",
-			},
-		}, "v2": {
-			{
-				Name:     "deployment-v2.yaml",
-				Content:  string(v2),
-				Encoding: "",
-			},
+func init() {
+	v1, _ := os.ReadFile(assetsPath + "/deployment-v1.yaml")
+	v2, _ := os.ReadFile(assetsPath + "/deployment-v2.yaml")
```

Comment:
No, the data needs to be there before the `BeforeSuite` in `suite_test.go`. 
I can move the preparation of the lookup mock into the `suite_test.go`, but test data wouldn't be in the same file as the test anymore. Or I can add a panic if there was an error loading the test data from the assets folder.

I don't care much about the error as it only occurs if a commit removes the test assets. However, then the test would fail a little later, so technically that commit shouldn't get merged...